boyzwhocried/bwc-next-ts
# Pull Request: Merge Dev into Staging

## Pull Request: Merge Dev into Staging

### Description

This pull request includes changes from the `dev` branch to the `staging` branch.

### Changes Made

- Merged changes from the `20-feature-request-swipeable-images-in-modal-detail-component` branch.

### Steps to Verify

1. Confirm successful merging of the `20-feature-request-swipeable-images-in-modal-detail-component` changes.
2. Verify any additional changes made on the `dev` branch.

### Screenshots

![image](https://github.com/boyzwhocried/bwc-next-ts/assets/71019245/2783ea9c-35f8-4f16-a492-0b900fa8302e)

![image](https://github.com/boyzwhocried/bwc-next-ts/assets/71019245/9f9686f5-a8ce-4cc3-803d-9b2f330dec3d)

### Environment

- Browser: Chrome for Android

### Additional Information

_(N/A)_

### Reviewers

- @boyzwhocried 

Please review the changes and provide feedback. Thank you!
